src/Docker.DotNet/Docker.DotNet.csproj

```diff
@@ -19,6 +19,7 @@
   </PropertyGroup>
   <ItemGroup>
     <PackageReference Include="Newtonsoft.Json" Version="9.0.1" />
+    <PackageReference Include="System.Runtime.InteropServices.RuntimeInformation" Version="4.3.0" />

```

Comment:
Shouldn't this package be included under a `Condition` to check the target framework for `net46` and `net45`?

Reply:
Looks like 4.3.0 package has built in support for net45/46/netstandard1.1 so it works to include it on all here.